com.unity.graphtools.foundation/Editor/UI/Views/RootView.cs

```diff
@@ -40,12 +40,13 @@ public abstract class RootView : VisualElement, IRootView
         /// </summary>
         /// <param name="window">The <see cref="EditorWindow"/> containing this view.</param>
         /// <param name="graphTool">The tool hosting this view.</param>
-        protected RootView(EditorWindow window, BaseGraphTool graphTool)
+        /// GTF-EDIT: Added parameter to constructor to allow for exposing dispatcher instance to use
+        protected RootView(EditorWindow window, BaseGraphTool graphTool, Dispatcher dispatcher = null)
         {
             focusable = true;
 
             GraphTool = graphTool;
-            Dispatcher = new CommandDispatcher();
+            Dispatcher = dispatcher ?? new CommandDispatcher();
```

Comment:
Did you verify w/GTF team that this doesn't violate flow/visibility/encapsulation rules?